projects/addon-charts/components/line-chart/line-chart.template.html

```diff
@@ -49,56 +49,61 @@
             [style.bottom.%]="getBottom(point[1])"
         ></div>
     </ng-container>
-    <ng-container *ngIf="hasHints">
-        <ng-container *ngFor="let point of value; let index = index">
-            <!-- TODO: Fix hint type -->
-            <div
-                *ngIf="value.length > 1 || dots"
-                class="t-column"
-                [class.t-column_hovered]="hovered === index"
-                [class.t-column_hint_hovered]="drivers.get(index) | async"
-                [tuiHintHost]="hintHost"
-                [tuiHint]="$any(hintDirective?.hint || hintContent)"
-                [tuiHintContext]="getContentContext(point, index)"
-                [tuiHintDescribe]="getHintId(index)"
-                [style.left.%]="getLeft(getX(index))"
-                [style.width.%]="getWidth(index)"
-                (mouseenter)="onMouseEnter(index)"
-            >
+    <ng-container *tuiLet="(axes?.focused$ | async) || !!(hints$ | async)?.length as axesFocused">
```

Comment:
Probably, we should merge it into in one stream inside component

```ts
someMeaningfulName$ = axes?.focused$.pipe(
    switchMap(...)
)
```

because:
1. it difficult to read it here in template
2. Too many subscriptions are produced (yep, `async`-pipe manages their unscribscription but I think it is not performance-friendly)

Reply:
fixed